tests/system/plugin.test.ts

```diff
@@ -238,29 +238,14 @@ test('inspect on mvn error', async (t) => {
 
 test('inspect on mvnw error', async (t) => {
   const targetFile = path.join(fixturesPath, 'bad-maven-with-mvnw', 'pom.xml');
-  const isWinLocal = /^win/.test(os.platform());
-  const mvnwCommand = isWinLocal ? `mvnw.cmd` : './mvnw';
-  const fullCommand = `${mvnwCommand} dependency:tree -DoutputType=dot --file="${targetFile}"`;
   try {
     await plugin.inspect('.', targetFile, {
       dev: true,
     });
     t.fail('expected inspect to throw error');
   } catch (error) {
-    const mvnwCommandTipMessage =
-      'Currently, you cannot run `mvnw` outside your current directory, you will have to go inside the directory of your project (see: https://github.com/takari/maven-wrapper/issues/133)\n\n';
     const expectedCommand =
-      '\n\n' +
-      'Please make sure that Apache Maven Dependency Plugin ' +
-      'version 2.2 or above is installed, and that `' +
-      fullCommand +
-      '` executes successfully ' +
-      'on this project.\n\n' +
-      mvnwCommandTipMessage +
-      'If the problem persists, collect the output of `' +
-      fullCommand +
-      '` and contact support@snyk.io\n';
-
+      '[WARNING] The POM for no.such.groupId:no.such.artifactId:jar:1.0.0 is missing, no dependency information available';
```

Comment:
Update test to assert actual maven error and not the wrapper invocation error (which wasn't supposed to be there)